modules/appnexusBidAdapter.js

```diff
@@ -598,6 +598,10 @@ function newBid(serverBid, rtbBid, bidderRequest) {
     bid.meta = Object.assign({}, bid.meta, { advertiserId: rtbBid.advertiser_id });
   }
 
+  if (rtbBid.brand_id) {
+    bid.meta = Object.assign({}, bid.meta, { brandId: rtbBid.brand_id });
```

Comment:
https://docs.prebid.org/dev-docs/bidder-adaptor.html specifies 'brandId' and 'advertiserId'

Reply:
I'm not sure I follow the question.  We hadd already specified advertiserId and set in the bid.meta object.  This PR adds support for brandId, which was missing.

Reply:
sorry I thought there was a typo that I was pointing out but my mistake, no typo.